cypress/integration/transfer/transferring-items.feature

```diff
@@ -0,0 +1,28 @@
+Feature: Transferring items between lists
+
+  Scenario: Transfer from options list (left) to selected list (right)
```

Comment:
I think we should omit the sides. Firstly, that's part of the design, not part of the behavior. Secondly, if this is important, it should be part of the steps, not of the scenario title (which won't be tested) and thirdly this can potentially change (if we add a mobile design, and we simply display the selected list below the options list but don't change any behavior, we should not have to change this feature file)

Reply:
Good point, fixed.